include/bluetooth/services/dfu_smp_c.rst

```diff
@@ -0,0 +1,77 @@
+.. _dfu_smp_c_readme:
+
+GATT DFU SMP Service Client
+###########################
+
+This module implements a Simple Management Protocol (SMP) Service Client that can be used in the context of Device Firmware Updates (DFU).
+SMP is a basic transfer encoding for use with the `MCUmgr`_ management protocol.
+See `SMP over Bluetooth`_ for the service specification.
+
+The SMP Client module can be used to interact with Zephyr's :ref:`zephyr:smp_svr_sample`.
+
+The SMP Client implements only the service.
+It does not provide any functionality to process or interpret SMP commands and responses.
```

Comment:
Is that what you meant with "The SMP Client here implements only simple service without any coding
and data interpretation." ?
And is it correct?

Reply:
LGTM